spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/trace/http/HttpTraceAutoConfiguration.java

```diff
@@ -34,11 +34,12 @@
  * {@link EnableAutoConfiguration Auto-configuration} for HTTP tracing.
  *
  * @author Dave Syer
+ * @author Artsiom Yudovin
  * @since 2.0.0
  */
 @Configuration
 @ConditionalOnWebApplication
-@ConditionalOnProperty(prefix = "management.trace.http", name = "enabled", matchIfMissing = true)
+@ConditionalOnProperty(prefix = "management.trace.http", name = "enabled")
```

Comment:
That effectively means the default value of this property is now `false`. This should be reflected in the metadata, the doc and the appendix